<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SunWidget</class>
 <widget class="QMainWindow" name="SunWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>351</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>351</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MPCV</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Sun position</string>
    </property>
    <widget class="QDoubleSpinBox" name="latitude">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>50.079999999999998</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>57</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="longitude">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>14.440000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>57</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>57</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="time">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>231</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>28</height>
      </size>
     </property>
     <property name="time">
      <time>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Render</string>
    </property>
    <widget class="QSpinBox" name="widthSpin">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="heightSpin">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>768</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>57</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Iterations</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="itersSpin">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="denoiseBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Denoise</string>
     </property>
    </widget>
   </widget>
   <action name="actionSave_render">
    <property name="text">
     <string>Save render</string>
    </property>
   </action>
   <action name="actionClose">
    <property name="text">
     <string>Close</string>
    </property>
   </action>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
